client/lib/components/learn_page_promo.dart

```diff
@@ -6,6 +8,22 @@ import 'package:who_app/components/promo_curved_background.dart';
 import 'package:who_app/components/themed_text.dart';
 import 'package:who_app/constants.dart';
 
+const double blurRadius = 20;
```

Comment:
Type is automatically inferred by appending `.0` on number:
```suggestion
const blurRadius = 20.0;
```